engine/src/main/java/nl/inl/blacklab/codec/TokensCodec.java

```diff
@@ -39,9 +39,10 @@ public static TokensCodec fromCode(byte code) {
     public enum VALUE_PER_TOKEN_PARAMETER {
         BYTE((byte) 0),
         SHORT((byte) 1),
-        INT((byte) 2);
+        THREE_BYTES((byte) 3),
+        INT((byte) 4);
```

Comment:
Why 0,1,3,4? Maybe better to either restrict it to 2 bits (0,1,2,3) or to use the number of bytes as the value (1,2,3,4) ?

Reply:
Woops!